proposals/threads/Overview.md

```diff
@@ -197,9 +197,9 @@ Both wake and wait operators trap if the effective address of either operator
 is misaligned or out-of-bounds. The wait operators requires an alignment of
 their memory access size. The wait operator requires an alignment of 32 bits.
 
-The embedder is also permitted to suspend or wake an agent. A suspended agent
-can be woken by the embedder or the wake operator, regardless of how the agent
-was suspended (e.g. via the embedder or a wait operator).
+For the web embedding, the agent can also be suspended or woken via the
+[`Atomics.wait`][] and [`Atomics.wake`][] functions respectively. An agent 
+will not be suspended or woken for other reasons.
```

Comment:
Right, as @lars-t-hansen mentions in #25, you can suspend an agent only if you suspend all agents in the cluster. I'll clarify.